### PR TITLE
feat: update logistration pages

### PR DESCRIPTION
**Pull request by zainab-amir**
- removed few unused classes
- updated login page to rely on error messages generated from the frontend only.

[VAN-138](https://2u-internal.atlassian.net/browse/VAN-138)

### CODE REVIEW

File: src/login/tests/LoginPage.test.jsx [hunk 1]
```diff
@@ -203,12 +204,12 @@ describe('LoginPage', () => {
...
 
   it('should match login error message', () => {
-    const errorMessage = 'Email or password is incorrect.';
+    const errorMessage = 'An error has occurred. Try refreshing the page, or check your internet connection.';
     store = mockStore({
       ...initialState,
       login: {
         ...initialState.login,
-        loginError: { value: errorMessage },
+        loginError: { errorCode: INTERNAL_SERVER_ERROR },
```
Comment (shafqatfarhan) [inline src/login/tests/LoginPage.test.jsx:212]: Should not we check here for `INCORRECT_EMAIL_PASSWORD = 'incorrect-email-or-password'` errorCode?

File: src/login/tests/LoginFailure.test.jsx [hunk 2]
```diff
@@ -202,10 +202,9 @@ describe('LoginFailureMessage', () => {
   });
 
   it('should match direct render of error message', () => {
```
Comment (shafqatfarhan) [inline src/login/tests/LoginFailure.test.jsx:204]:
```suggestion
  it('should match internal server error message', () => {
```

**Review by shafqatfarhan — APPROVED** [review]
Left a few suggestions, other wise looks good 👍

File: src/forgot-password/ForgotPasswordPage.jsx [hunk 3]
```diff
@@ -57,16 +58,16 @@ const ForgotPasswordPage = (props) => {
   };
 
   const tabTitle = (
-    <div className="d-flex">
-      <Icon src={ChevronLeft} className="arrow-back-icon" />
+    <div className="d-inline-flex flex-wrap align-items-center">
```
[the remark this answers is not in the document]
Comment (zainab-amir) [reply src/forgot-password/ForgotPasswordPage.jsx:61]: It is according to design.

**Pull request merged**